### ISSUE TITLE
Delay module time (ms) control on rotary

### ISSUE THREAD

**Issue by pbuffet** [polyeffects/digit_ui#1]
Could it be possible to adjust the time on the delay module with the left (fine adjustment knob) by 1 ms increments.
As the gab between value make it not usable friendly.

Maybe the second knob can go in 200 ms increments

Don't hesitate to move the issue in the proper area.
Regards,
PB

**Comment by loki42** [comment]
good idea but not currently on the todo. You can control warp for fine changes.
